pkg/api/chunk_test.go

```diff
@@ -118,6 +120,21 @@ func TestChunkUploadDownload(t *testing.T) {
 		}
 
 	})
+	t.Run("retrieve-targets", func(t *testing.T) {
+		jsonhttptest.ResponseDirect(t, client, http.MethodPost, resource(validHash), bytes.NewReader(validContent), http.StatusOK, jsonhttp.StatusResponse{
+			Message: http.StatusText(http.StatusOK),
+			Code:    http.StatusOK,
+		})
+
+		// try to fetch the same chunk
+		resp := request(t, client, http.MethodGet, resourceTargets(validHash), nil, http.StatusOK)
+
+		// Check if the target is obtained correctly
+		if resp.Header.Get("Targets") != targets {
+			t.Fatalf("targets mismatch. got %s, want %s", resp.Header.Get("Targets"), targets)
+		}
+
+	})
```

Comment:
what do you think if we were to do something like this here?

```suggestion
	t.Run("retrieve-targets", func(t *testing.T) {
		resp := request(t, client, http.MethodGet, resourceTargets(validHash), nil, http.StatusOK)

		// Check if the target is obtained correctly
		if resp.Header.Get("Targets") != targets {
			t.Fatalf("targets mismatch. got %s, want %s", resp.Header.Get("Targets"), targets)
		}
	})
```

Reply:
I left it with this implementation